widgets/range-slider/range-slider.js

```diff
@@ -95,6 +94,7 @@ function rangeSlider({
         templates
       });
 
+      var Slider = autoHide(headerFooter(require('../../components/Slider/Slider')));
```

Comment:
This is why moving them up in the tree (the decorators) should be what we need to do, because while testing we emphasis that the behavior of the widget is a lot dependant of the decorators. We need "pure" widgets :)